setup.py

```diff
@@ -26,7 +26,11 @@ def load_requirements(use_case):
     install_requires=load_requirements("app"),
     setup_requires=["setuptools_scm"],
     use_scm_version={"local_scheme": "no-local-version"},  # handled by setuptools_scm
-    packages=find_packages(),  # will include *.py files and some other types
+    packages=find_packages()
+    + [
+        "flexmeasures.ui.templates",
+        "flexmeasures.ui.static",
+    ],  # will include *.py files and some other types
```

Comment:
I don't see the effect of this addition. It looks like with or without it, we get warnings about these packages being absent from the \`packages\` configuration, **and** they also still end up in the .tar.gz file.

I also saw that without `include_package_data=True`, we get no such warnings, **and** these packages also still end up in the .tar.gz file.

Reply:
For me, this stopped the warnings.

I used. /to_pypi.sh --dry-run